docs/README.md

```diff
@@ -0,0 +1,20 @@
+[![ReadTheDocs][doc-img]][doc]
+
+# Jaeger Documentation
+
+This directory contains Jaeger documentation hosted by [readthedocs][doc].
+
+## Building
+
+The documentation is built with [MkDocs](http://www.mkdocs.org/).
+You need to have [virtualenv](https://virtualenv.pypa.io/en/stable/) installed.
+
+Then simply run `make docs`.
+
+## Deploying
+
+Raise a PR on GitHub. Once the PR is approved and merged,
+ask one of Jaeger maintainers to kick off the build on [readthedocs][doc].
```

Comment:
I changed the link, but I don't believe that "anyone" can kick off a build, you need to be authorized for the project on read-the-docs.